### PR TITLE
Implement Bandit CI Tests

### PR DESCRIPTION
**Pull request by lukehinds**
Resolves: #51

Signed-off-by: Luke Hinds <lhinds@redhat.com>

### CODE REVIEW

**Comment by lukehinds** [comment]
So we can see a failure on this, there are two ways of dealing with false postives.

One is to append nosec e.g. ` some_code # nosec`

Or if you prefer to not run a particular test, we can add the test number (which in the failed case would be `B102`: https://bandit.readthedocs.io/en/1.7.4/plugins/b102_exec_used.html

**Comment by woodruffw** [comment]
> Or if you prefer to not run a particular test, we can add the test number (which in the failed case would be `B102`: https://bandit.readthedocs.io/en/1.7.4/plugins/b102_exec_used.html

Yeah, let's exclude this specific test on that specific result: it's an intentional use of `exec`.

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 version = {}
 with open("./sigstore/_version.py") as f:
-    exec(f.read(), version)
+    exec(f.read(), version) # nosec
```
Comment (woodruffw) [inline setup.py:21]:
```suggestion
    exec(f.read(), version) # nosec B102
```

**Comment by woodruffw** [comment]
LGTM modulo restricting the `nosec`!

**Review by woodruffw — APPROVED** [review]

**Pull request merged**